### PR TITLE
Gather changes for shapes, requests, and responses

### PR DESCRIPTION
**Pull request by smizell**
Add resolvers to get changes for each shape ID.

### CODE REVIEW

File: workspaces/graph-lib/src/shapes-graph/index.ts [hunk 1]
```diff
@@ -289,7 +319,7 @@ export class GraphQueries {
     throw new Error(`unexpected node.type`);
   }
 
-  wrapList(type: NodeType, nodes: Node[]): NodeListWrapper {
+  wrapList(type: NodeType | null, nodes: Node[]): NodeListWrapper {
```
Comment (smizell) [inline workspaces/graph-lib/src/shapes-graph/index.ts:322]: I'm adding a TODO to move away from this at some point soon.

**Review by devdoshi — APPROVED** [review]
looks good to me, including your future plans. it's a little hard to review without the test cases/snapshots because there are a lot of edge cases but overall looks right
